tests/linalg/test_matrix.py

```diff
@@ -605,24 +607,47 @@ def test_normalized_hypergraph_laplacian():
 
     assert isinstance(L2, np.ndarray)
     assert np.all(L1.toarray() == L2)
-    assert np.all(np.diag(L2) == 0.5)
 
-    L3, d = xgi.normalized_hypergraph_laplacian(H, index=True)
+    evals = eigh(L2, eigvals_only=True)
```

Comment:
if you don't need the eigenvalues, you can use `eigvalsh`, which outputs them in ascending order. I think it's supposed to be faster